### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by Dioni1195**

### CODE REVIEW

File: api/v1/views/cities.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    if catch_city is None:
+        abort(404)
+    storage.delete(catch_city)
+    storage.save()
+    return jsonify({}), 200
+
+
+@app_views.route('/states/<state_id>/cities',
+                 methods=['POST'], strict_slashes=False)
+def post_cities(state_id):
```
Comment (mrdoomus) [inline api/v1/views/cities.py]: I saw in other scripts that this definition could use a better name. Something like "post_state_id_cities" It makes more sense regarding the route.

File: api/v1/views/cities.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+from api.v1.views import app_views
+from flask import Flask, jsonify, abort, make_response, request
+from models import storage
+from models.state import State
+from models.city import City
+
+
+@app_views.route('/states/<state_id>/cities',
+                 methods=['GET'], strict_slashes=False)
+def get_cities_states_id(state_id):
```
Comment (mrdoomus) [inline api/v1/views/cities.py]: saw in other scripts that this definition could use a better name. Something like "get_state_id_cities" It makes more sense regarding the route.

File: api/v1/views/places.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+from api.v1.views import app_views
+from flask import Flask, jsonify, abort, request
+from models import storage
+from models.place import Place
+from models.city import City
+
+
+@app_views.route('/cities/<city_id>/places',
+                 methods=['GET'], strict_slashes=False)
+def get_places_cities_id(city_id):
```
Comment (mrdoomus) [inline api/v1/views/places.py]: saw in other scripts that this definition could use a better name. Something like "get_cities_id_places"

File: api/v1/views/places.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    if catch_place is None:
+        abort(404)
+    storage.delete(catch_place)
+    storage.save()
+    return jsonify({}), 200
+
+
+@app_views.route('/cities/<city_id>/places',
+                 methods=['POST'], strict_slashes=False)
+def post_places(city_id):
```
Comment (mrdoomus) [inline api/v1/views/places.py]: saw in other scripts that this definition could use a better name. Something like "post_cities_id_places"

File: api/v1/views/places_reviews.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+from api.v1.views import app_views
+from flask import Flask, jsonify, abort, request
+from models import storage
+from models.place import Place
+from models.review import Review
+
+
+@app_views.route('/places/<place_id>/reviews',
+                 methods=['GET'], strict_slashes=False)
+def get_places_reviews_id(place_id):
```
Comment (mrdoomus) [inline api/v1/views/places_reviews.py]: saw in other scripts that this definition could use a better name. Something like "get_places_id_reviews"

File: api/v1/views/places_reviews.py [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+    if catch_review is None:
+        abort(404)
+    storage.delete(catch_review)
+    storage.save()
+    return jsonify({}), 200
+
+
+@app_views.route('/places/<place_id>/reviews',
+                 methods=['POST'], strict_slashes=False)
+def post_reviews(place_id):
```
Comment (mrdoomus) [inline api/v1/views/places_reviews.py]: saw in other scripts that this definition could use a better name. Something like "post_places_id_reviews"

**Review by mrdoomus — CHANGES_REQUESTED** [review]
For the api folder to have cohesion, I recommend changing those definition names.

File: api/v1/views/cities.py (hunk 1 above)
Comment (Dioni1195) [reply api/v1/views/cities.py]: Ok, I agree. Change fixed

File: api/v1/views/cities.py (hunk 2 above)
Comment (Dioni1195) [reply api/v1/views/cities.py]: Change fixed

File: api/v1/views/places.py (hunk 3 above)
Comment (Dioni1195) [reply api/v1/views/places.py]: Change fixed

File: api/v1/views/places.py (hunk 4 above)
Comment (Dioni1195) [reply api/v1/views/places.py]: Change fixed

File: api/v1/views/places_reviews.py (hunk 5 above)
Comment (Dioni1195) [reply api/v1/views/places_reviews.py]: Ok, change fixed

File: api/v1/views/places_reviews.py (hunk 6 above)
Comment (Dioni1195) [reply api/v1/views/places_reviews.py]: Change fixed

**Review by mrdoomus — APPROVED** [review]
Ok! Great job

**Pull request merged**